playwright-e2e/page-objects/local-authority/birth-father/laBirthFatherParentalResponsibilityPage.page.ts

```diff
@@ -0,0 +1,25 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LABirthFatherParentalResponsibilityPage extends BasePage {
+  readonly birthFatherParentalResponsibilityHeader: Locator;
+
+  readonly yesRadioButton: Locator;
+  readonly noRadioButton: Locator;
+
+  readonly saveAndConinueButton: Locator;
+  readonly saveAsDraftButton: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    this.birthFatherParentalResponsibilityHeader = page.getByRole('heading', { name: 'Does the birth father have' });
+
+    this.yesRadioButton = page.getByLabel('Yes');
+    this.noRadioButton = page.getByLabel('No');
+
+    this.saveAndConinueButton = page.getByRole('button', { name: 'Save and continue' });
+    this.saveAsDraftButton = page.getByRole('button', { name: 'Save as draft' });
+  }
+}
```

Comment:
I can't see a page for if they select No Parental Responsibility on this page --> la-portal/birth-father/parental-responsibility/no-responsibility

Reply:
Good point, for now, as the happy path was decided to not include the page for no-responsibility, this page/fixture will not be implemented yet. This will be introduced in a later Jira.

Reply:
@AustenStevensMOJ if the jira already exists this can be resolved. If not it's worth raising now and putting on the backlog so it's not forgotten.